### PR TITLE
Add aria-hidden="true" => yii\grid\ActionColumn

### PR DESCRIPTION
**Pull request by lav45**
http://getbootstrap.com/components/#accessible-icons

### CODE REVIEW

File: framework/grid/ActionColumn.php [hunk 1]
```diff
@@ -110,7 +110,7 @@ protected function initDefaultButtons()
     {
         if (!isset($this->buttons['view'])) {
             $this->buttons['view'] = function ($url, $model, $key) {
-                return Html::a('<span class="glyphicon glyphicon-eye-open"></span>', $url, array_merge([
+                return Html::a('<span class="glyphicon glyphicon-eye-open" aria-hidden="true"></span>', $url, array_merge([
```
Comment (samdark) [inline framework/grid/ActionColumn.php]: Won't it completely hide the button from screen readers?

**Comment by cebe** [comment]
@LAV45 you should read the full explaination of what you linked. For this case, the following applies instead: 

> If you're creating controls with no other text (such as a <button> that only contains an icon), you should always provide alternative content to identify the purpose of the control, so that it will make sense to users of assistive technologies. In this case, you could add an aria-label attribute on the control itself.

**Comment by LAV45** [comment]
Excuse me, didn't read :)

**Comment by cebe** [comment]
can you apply the correct changes?

**Comment by samdark** [comment]
Should be aria-label.

**Comment by samdark** [comment]
Merged with some adjustments. Thanks!

**Comment by samdark** [comment]
@LAV45 it's better to use the same email for commits as you're using for GitHub so commit and profile will be linked together.

File: framework/grid/ActionColumn.php [hunk 2]
```diff
@@ -110,28 +110,31 @@ protected function initDefaultButtons()
     {
         if (!isset($this->buttons['view'])) {
             $this->buttons['view'] = function ($url, $model, $key) {
-                return Html::a('<span class="glyphicon glyphicon-eye-open"></span>', $url, array_merge([
+                $options = array_merge([
                     'title' => Yii::t('yii', 'View'),
                     'data-pjax' => '0',
-                ], $this->buttonOptions));
+                ], $this->buttonOptions);
+                return Html::a('<span class="glyphicon glyphicon-eye-open" aria-label="' . $options['title'] . '"></span>', $url, $options);
```
Comment (qiangxue) [inline framework/grid/ActionColumn.php]: `$options['title']` should be HTML-encoded. Similar thing for the code below.

**Comment by LAV45** [comment]
View, Update and Delete - it's a simple word in which no characters need to be escaped using HTML-encode.

**Comment by cebe** [comment]
it is translated by Yii::t and thus may be anything.

**Comment by samdark** [comment]
What was merged is *adjusted* version ;)

**Comment by cebe** [comment]
good point :-) But shouldn't the label be on the span rather than the link?

**Comment by samdark** [comment]
Logically it should be on the link.

**Pull request closed without merging**